packages/pds/src/auth.ts

```diff
@@ -234,5 +233,28 @@ export const adminVerifier =
   }
 
 export const getRefreshTokenId = () => {
-  return uint8arrays.toString(crypto.randomBytes(32), 'base64')
+  return ui8.toString(crypto.randomBytes(32), 'base64')
+}
+
+export const createServiceJwt = async (
+  accountDid: string,
+  keypair: crypto.Keypair,
+): Promise<string> => {
+  const header = {
+    typ: 'JWT',
+    alg: keypair.jwtAlg,
+  }
+  const exp = Math.floor((Date.now() + MINUTE) / 1000)
+  const payload = {
+    iss: accountDid,
+    exp,
+  }
+  const toSignStr = `${jsonToB64Url(header)}.${jsonToB64Url(payload)}`
+  const toSign = ui8.fromString(toSignStr, 'utf8')
+  const sig = await keypair.sign(toSign)
+  return `${toSignStr}.${ui8.toString(sig, 'base64url')}`
+}
+
+const jsonToB64Url = (json: Record<string, unknown>): string => {
+  return ui8.toString(ui8.fromString(JSON.stringify(json), 'utf8'))
```

Comment:
Same here, I feel like I'm missing where `base64url` comes into play—I think `toString()` and `fromString() defaults to encoding to `utf8`, so what I expected to see is more like this:
```suggestion
  return ui8.toString(ui8.fromString(JSON.stringify(json)), 'base64url')
```

Reply:
yup yup 👍 

altho it's actually
`return ui8.toString(ui8.fromString(JSON.stringify(json), 'utf8'), 'base64url')`

i'm gonna split these into helper methods to be less confusing